### ISSUE TITLE
BUG: Adding the algo

### ISSUE THREAD

**Issue by Mansi-Tanwar** [sakeel-103/DFS-BFS-Graph-Travers#486]
### Preliminary Check

- [X] I have searched the existing issues
- [X] This issue is reproducible

### Bug Summary

![image](https://github.com/user-attachments/assets/3962c1fd-577c-4ee5-ae7b-5c2103976664)

The algo will be better understandable if along with the visual representation we have it's algo also. So I would love to add the pseudocode of the algorithm, if you assign this to me under hacktoberfest. Thank You. 

### Steps to Reproduce

I will add the pseudocode of the algorithm on the page where the visual representation is shown. 

### Additional Information

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue
- [X] I can provide more details if needed

**Comment by sakeel-103** [comment]
@Mansi-Tanwar 
Please check the graphs already have visualizer

![Screenshot 2024-10-30 200716](https://github.com/user-attachments/assets/ae0a209d-2c69-4b80-983d-d5dbdacfe4c6)

Please check it once bcz there are more algo's added

**Comment by Mansi-Tanwar** [comment]
@sakeel-103 You said to check the graphs already have a visualizer. I didn't get what you said. Are you asking me to check if visualiser is working properly??? or what? can you please explain.

**Comment by sakeel-103** [comment]
> @sakeel-103 You said to check the graphs already have a visualizer. I didn't get what you said. Are you asking me to check if visualiser is working properly??? or what? can you please explain.

@Mansi-Tanwar 
yes, this was I meant check the visualizer first and then work accordingly
